storage/sql/sql.go

```diff
@@ -22,6 +22,10 @@ type flavor struct {
 	// Optional function to create and finish a transaction.
 	executeTx func(db *sql.DB, fn func(*sql.Tx) error) error
 
+	// Function for locking rows. It must return error if locking failed. If underlying
```

Comment:
This comment needs to be expanded to explain what `value` is and why it's required.